docker_scripts/config-docker-sat.sh

```diff
@@ -30,7 +30,7 @@ SATMANDIR=/usr/share/man/man8
 METAL_PROVISION_REPO="https://github.com/Cray-HPE/metal-provision.git"
 METAL_PROVISION_DIR="metal-provision"
 METAL_PROVISION_BASE_PACKAGES_PATH="roles/node_images_ncn_common/vars/packages/suse.yml"
-METAL_PROVISION_BRANCH="main"
+METAL_PROVISION_BRANCH="lts/csm-1.5"
```

Comment:
We're going to make this change in integration and main, so that we don't have to do it each time we cut a new release branch in this repository. We will just have to remember to update it each time we create a new SAT product release that targets a new version of CSM. Jack is documenting this in our branching model page.